### PR TITLE
Added NuGet Package for .Net Framework target.

### PR DESCRIPTION
**Pull request by barnstee**
NuGet package contains everything you need to design & build a .Net-based GW module. Things that still need to be fixed in the SDK:
1. Build all .pdb files for DLLs in Release configuration
2. Build nanomsg DLL in Release configuration

### CODE REVIEW

File: nuget/Azure.IoT.Gateway.SDK.nuspec [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Azure.IoT.Gateway.SDK.Net</id>
+    <version>2017.01.13.2</version>
```
Comment (tameraw) [inline nuget/Azure.IoT.Gateway.SDK.nuspec]: >2017.01.13.2 [](start = 13, length = 12)

Investigate if this is supported by Nuget. I am not sure, and don't recall seeing this in packages I worked with.

Comment (barnstee) [inline nuget/Azure.IoT.Gateway.SDK.nuspec]: Yeah, I think it's needs to be 2017.01.13-2, checking...

Comment (barnstee) [inline nuget/Azure.IoT.Gateway.SDK.nuspec]: https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#semantic-versioning
So it needs to be 2017.01.13-alpha

**Comment by zafields** [comment]
Address #96 and fixes the updates the build_nodejs.cmd to properly create the environment variables for the subsequent step on the node simple sample how to

**Comment by damonbarry** [comment]
There's a much larger effort in process right now to produce gateway SDK packages for various platforms & languages, starting with NuGet/.NET. This will include our engineering process for publishing packages, so the actual .nuspec file is a relatively minor detail.

Since this PR has stalled a bit, we're going to close it. The work that was already scheduled (just getting underway now) will continue. Thanks!

**Pull request closed without merging**